src/components/IOUConfirmationList.js

```diff
@@ -102,21 +112,66 @@ const defaultProps = {
     comment: '',
     network: {},
     myPersonalDetails: {},
+    iouType: CONST.IOU.IOU_TYPE.REQUEST,
 };
 
 class IOUConfirmationList extends Component {
     constructor(props) {
         super(props);
 
-        this.toggleOption = this.toggleOption.bind(this);
-
         const formattedParticipants = _.map(this.getParticipantsWithAmount(this.props.participants), participant => ({
             ...participant, selected: true,
         }));
 
+        // Add the button options to payment menu
+        const confirmationButtonOptions = [];
+        let defaultButtonOption = {
+            text: this.props.translate(this.props.hasMultipleParticipants ? 'iou.split' : 'iou.request', {
+                amount: this.props.numberFormat(
+                    this.props.iouAmount,
+                    {style: 'currency', currency: this.props.iou.selectedCurrencyCode},
+                ),
+            }),
+        };
+        if (this.props.iouType === CONST.IOU.IOU_TYPE.SEND && this.props.participants.length === 1) {
+            // Add the Expensify Wallet option if available and make it the first option
+            if (this.props.localCurrencyCode === CONST.CURRENCY.USD || Permissions.canUsePayWithExpensify(this.props.betas)) {
+                confirmationButtonOptions.push({text: this.props.translate('iou.settleExpensify'), icon: Wallet});
+            }
+
+            // Add PayPal option
+            if (this.props.participants[0].payPalMeAddress) {
+                confirmationButtonOptions.push({text: this.props.translate('iou.settlePaypalMe'), icon: PayPal});
+            }
+            defaultButtonOption = {text: this.props.translate('iou.settleElsewhere'), icon: Cash};
```

Comment:
I'm a little confused by this, so the options are going to be `settle elsewhere` if there is one participant, otherwise it will say `split`?

Reply:
Or is it that send is an entirely different flow?

Reply:
> Or is it that send is an entirely different flow?

Send is an entirely different flow! We have an array of options to send money with, but only one if you're splitting or requesting